brianlong/www.validators.app
# 186655494 validator index with skipped after

#### What's this PR do?
- add skipped after to validators index

#### How should this be manually tested?
- run gather and score daemons
- check validators table for new metric

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186655494)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
